lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/AllowAllSystemAccessControl.java

```diff
@@ -359,6 +359,9 @@ public Map<ColumnSchema, ViewExpression> getColumnMasks(SystemSecurityContext co
         return ImmutableMap.of();
     }
 
+    @Override
+    public void checkCanSetEntityAuthorization(SystemSecurityContext context, String ownedKind, List<String> name, TrinoPrincipal principal) {}
```

Comment:
We can remove the other `checkCanSet*Authorization` methods from this impleentation and all others, since the engine can never call the other  versions (because they are marked for removal)

Reply:
Removed.